ui/draggable.js

```diff
@@ -141,14 +141,20 @@ $.widget("ui.draggable", $.ui.mouse, {
 	},
 
 	_blurActiveElement: function( event ) {
-
-		// Only need to blur if the event occurred on the draggable itself, see #10527
-		if ( !this.handleElement.is( event.target ) ) {
+		var activeElement = $.ui.safeActiveElement( this.document[ 0 ] ),
+			target = $( event.target );
+
+		// Only blur if the event occurred on an element that is:
+		// 1) within the draggable handle
+		// 2) but not within the currently focused element
+		// See #10527, #12472
+		if ( !target.closest( this.handleElement ).length ||
```

Comment:
I think it'd be a good idea to change this condition to

```js
if ( this._getHandle(event) && target.closest( activeElement ).length )
```

While this patch works, it still duplicates the logic of "did this event occur within a handle" which opens the possibility of future bugs if the two different implementations aren't kept in sync